clients/lsp-eslint.el

```diff
@@ -36,6 +36,18 @@
   :group 'lsp-mode
   :link '(url-link "https://github.com/Microsoft/vscode-eslint"))
 
+(defcustom lsp-eslint-zip-path (f-join lsp-server-install-dir "eslint/extension.zip")
```

Comment:
Is this needed? I think we should just keep the `zip` file as temp file. There's no value customizing it.

Reply:
It can go away, I will update the PR.

Reply:
Fixed now.